xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1613,11 +1607,6 @@ int CGUIWindowVideoBase::GetScraperForItem(CFileItem *item, ADDON::ScraperPtr &i
   return foundDirectly ? 1 : 0;
 }
 
-void CGUIWindowVideoBase::OnScan(const std::string& strPath, bool scanAll)
```

Comment:
Why remove this? Now there are two places in the code that both need to call `g_application.StartVideoScan()` manually.